sql/sql_table.cc

```diff
@@ -3592,6 +3593,28 @@ mysql_prepare_create_table(THD *thd, HA_CREATE_INFO *create_info,
   {
     const CHARSET_INFO *save_cs;
 
+    /* check if the column is compressible.*/
+    if (sql_field->sql_type == MYSQL_TYPE_TINY_BLOB ||
+        sql_field->sql_type == MYSQL_TYPE_MEDIUM_BLOB ||
+        sql_field->sql_type == MYSQL_TYPE_BLOB ||
+        sql_field->sql_type == MYSQL_TYPE_LONG_BLOB ||
+        sql_field->sql_type == MYSQL_TYPE_VARCHAR)
+    {
+      DBUG_EXECUTE_IF("enforce_all_compressed_columns",
```

Comment:
Please do a enforce_all_compressed_columns / remove_compressed_attributes_for_keys run if you haven't